helpers/mathjax.js

```diff
@@ -77,6 +77,10 @@ export class HtmlBlockMathRenderer {
 
 }
 
+export function createHtmlBlockRenderer() {
```

Comment:
Given the new way we want to do renderer registration, I figured it would be nice if all of the renderer imports had a consistent API that could be used to instantiate the renderer. That way the provider can do something generic like:
```
import(path).then(renderer => renderer.createHtmlBlockRenderer())
```

Reply:
I see, so that's why the exported methods are all named the same.

Reply:
Yeah. I don't feel super strongly about this. The alternative is that the provider would need to do something like:
```
import(path).then(renderer => new renderer.WhateverTheRendererClassIs())
```
I figured the way I did it was marginally cleaner, but I could see benefits to both. Needing to alias the default imports is certainly a bit cumbersome.

Reply:
So the loader module would look something like...?
```javascript
provideInstance(document, 'html-block-renderer-loader', {
	getRenderers() {
		return Promise.all(
			['rendererPath1', `rendererPath2`, ...].map(path => import(path).then(renderer => renderer.createHtmlBlockRenderer()))
		);
	}
});
```
vs.
```javascript
provideInstance(document, 'html-block-renderer-loader', {
	getRenderers() {
		return Promise.all([
			import('rendererPath1').then(renderer => renderer.WhateverTheRenderer1MethodIs()),
			import('rendererPath2').then(renderer => renderer.WhateverTheRenderer2MethodIs()),
			...
		]);
	}
});
```
Yeah it's marginally cleaner. The aliasing of the imports is only something that needs to be done for the default renderers that html-block references directly. Unless I am missing something.

Also, I _think_ that import chain in the first case would resolve to the renderers, but I'm not 100% sure. Either way, looking good on the html-block end of things.

Reply:
Yeah, what I have mocked up in my local provider looks like this:
```
provideInstance(document, 'html-block-renderer-loader', {
	_renderers: [
		() => import('rendererPath').then(renderer => renderer.createHtmlBlockRenderer())
	],
	getRenderers() {
		return Promise.all(this._renderers.map(action => action()));
	}
});
```
The structure is a bit strange, but the `import` statements need to live outside of `getRenderers` or the scope will be wrong and they won't resolve properly. It's a bit of a weird structure but it works and does keep things deferred. 😆 

And yeah, I've confirmed locally that this does work as expected, so in terms of functionality I think we're good.